packages/ui/src/UserAvatar/UserAvatar.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+
+import { CustomerBase } from "@eisbuk//shared";
+
+interface UserAvatarProps extends Pick<CustomerBase, "name" | "surname"> {
+  avatar?: string;
+}
+
+/**
+ * Returns initials from provided name and last name
+ * @param name
+ * @param surname
+ * @returns
+ */
+const getInitials = (name: string, surname: string): string =>
+  `${name[0]}${surname[0]}`;
+
+const UserAvatar: React.FC<UserAvatarProps> = ({ name, surname, avatar }) => {
+  const [avatarUrl, setAvatarUrl] = useState(avatar);
+
+  const containerClassNames = containerClasses.join(" ");
+  const userNameClassNames = userNameClasses.join(" ");
+  const avatarClassNames = avatarClasses.join(" ");
+
+  return (
+    <div className={containerClassNames}>
+      <div>
+        <p className={userNameClassNames}>
+          {name} {surname}
+        </p>
+      </div>
+      <div className={avatarClassNames}>
+        {avatarUrl ? (
+          <img
+            className="rounded-full object-cover"
+            src={avatarUrl}
+            onError={({ currentTarget }) => {
+              currentTarget.onerror = null;
+              setAvatarUrl(undefined);
+            }}
```

Comment:
Intersting...could you provide the resource for this (I was completely unaware of this)

Reply:
Same here.

I found this [SO: Replace image source on error](https://stackoverflow.com/questions/34097560/react-js-replace-img-src-onerror), and the [MDN Docs: error event](https://developer.mozilla.org/en-US/docs/Web/API/Element/error_event). It is apparently a global element attribute.